static/js/common/upload-addon.js

```diff
@@ -327,12 +327,28 @@
 
                     $("<strong>").text(message).appendTo(upload_results);
 
+                    // Specific messages for unlisted addons.
+                    var isListed = $('#id_is_listed').is(':checked'),
+                        isSideload = $('#id_is_sideload').is(':checked');
+                    if (!isListed) {
+                      if (isSideload) {
+                        $("<p>").text(gettext("Your submission, as a sideloaded add-on, will go through a manual review.")).appendTo(upload_results);
+                      } else {
+                        if (warnings === 0) {
+                          $("<p>").text(gettext("Your submission passed validation and will therefore be automatically signed.")).appendTo(upload_results);
+                        } else {
+                          $("<p>").text(gettext("Your submission didn't pass validation, which is a requirement for automatic signing. It will therefore go through a manual review.")).appendTo(upload_results);
```

Comment:
Unlisted add-ons that don't pass validation will get the option to be added to the manual review queue. That's why there's a prompt in the flow chart. They shouldn't be queued automatically.